queries.sql

```diff
@@ -29,4 +29,13 @@ SELECT * from animals WHERE name NOT IN ('Gabumon');
 
 -- Find all animals with a weight between 10.4kg and 17.3kg inclusive
 
-SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- ################################# DAY 2 ######################################
+
+SELECT count(id) FROM animals;
+SELECT count(id) FROM animals WHERE escape_attempts = 0;
+SELECT avg(weight_kg) from animals;
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
```

Comment:
- [ ] Kindly be noticed that the question says `who escapes the most, neutered or not neutered animals?` not `who has the most escapes, neutered or not neutered?`.
```sql
SELECT neutered, MAX(escape_attempts) FROM animals
  GROUP BY neutered;
```

Reply:
Oh now I understand thank you. I had misinterpreted that.